Configuration/Eras/python/Era_Run2_2017_trackingLowPU_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+from Configuration.Eras.Modifier_trackingLowPU_cff import trackingLowPU
+
+Run2_2017_trackingLowPU = cms.ModifierChain(Run2_2017, trackingLowPU)
```

Comment:
@slava77 It will need a change, which will conflict with changing it to `run2_2017_core`. Because of the coupling, I agree it's easiest to leave as it is (since it wll be fixed soon).

Other thought: if `trackingLowPU` configuration should be kept working also in 2017, it would probably be good to add a matrix workflow for it, right?